cypress/integration/learn/challenges/output.js

```diff
@@ -59,3 +62,25 @@ describe('Classic challenge', function () {
       });
   });
 });
+
+describe('jQuery challenge', function () {
+  before(() => {
+    cy.visit(locations.jQuery);
+  });
+
+  it('renders the default output text', () => {
+    cy.title().should(
+      'eq',
+      'jQuery: Target HTML Elements with Selectors Using jQuery | freeCodeCamp.org'
+    );
+    cy.get(selectors.defaultOutput).contains(defaultOutput);
+  });
+
+  it('should not show a reference error', () => {
+    cy.wait(5000);
+    cy.get(selectors.defaultOutput).should(
+      'not.contain',
+      'ReferenceError: $ is not defined'
```

Comment:
This error is the same across Firefox and Chrome? I realise we Cypress for both, but I wonder if it is too reliant on one.

Reply:
FWIW, this is a bit of a redundancy as it first looks for the default text, which shouldn't display if the code throws an error.

I can verify the error message on firefox though. Will confirm shortly.

Reply:
Firefox:

![image](https://user-images.githubusercontent.com/63889819/145270980-d7215eae-999a-4732-a3ca-ae4e660f4d3d.png)


Chrome:

![image](https://user-images.githubusercontent.com/63889819/145270998-363e6018-967f-4808-8d36-cb1d42963216.png)

Reply:
FYI the default text will show briefly since the preview is rendered asynchronously.